code/game/objects/items/weapons/tools.dm

```diff
@@ -145,25 +140,20 @@
 	pickup_sound = 'sound/items/pickup/wirecutter.ogg'
 	var/bomb_defusal_chance = 30 // 30% chance to safely defuse a bomb
 	var/build_from_parts = TRUE
-	var/tool_colors = list(
-		"blue" = COLOR_BLUE,
-		"red" = COLOR_RED,
-		"purple" = COLOR_PURPLE,
-		"brown" = COLOR_BROWN,
-		"green" = COLOR_GREEN,
-		"cyan" = COLOR_CYAN,
-		"yellow" = COLOR_YELLOW
-	)
 
 /obj/item/wirecutters/Initialize()
 	. = ..()
 	if(build_from_parts)
-		var/our_color = pick(tool_colors)
-		var/color_hex = tool_colors[our_color]
-		color = color_hex
-		item_state = "[initial(icon_state)]-[our_color]"  // hardcoded. sucks, but inhands are hard and I can't be bothered.
+		color = pick(COLOR_BLUE, COLOR_RED, COLOR_PURPLE, COLOR_BROWN, COLOR_GREEN, COLOR_CYAN, COLOR_YELLOW)
 		add_overlay(overlay_image(icon, "[initial(icon_state)]_head", flags=RESET_COLOR))
 
+/obj/item/wirecutters/worn_overlays(icon_file)
+	. = list()
+	if(build_from_parts)
+		var/mutable_appearance/M = mutable_appearance(icon_file, "wirecutters_head")
+		M.appearance_flags = RESET_COLOR
+		. += M
```

Comment:
this looks like a lot of copypaste that you can simplify by just using the parent proc and parametrizing it